docs/dev/index.mdx

```diff
@@ -67,7 +67,7 @@ Clarification and discussion about key concepts, architecture, and development s
     - [Testing web code](/dev/background-information/testing_web_code)
 - [Building p4-fusion](/dev/background-information/build_p4_fusion)
 - [The `gitserver` API](/dev/background-information/gitserver-api)
-- [Using gRPC alongside REST for internal APIs](/dev/background-information/gRPC_internal_api)
+- [gRPC Tutorial](/dev/background-information/grpc_tutorial)
```

Comment:
Not actually related to this PR, but it'd be nice if there were more structure here, for example having a section for all things related to "backend platform" development.

Reply:
We can revisit this in a separate change